extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -67,6 +67,36 @@ export class AmpAudio extends AMP.BaseElement {
     this.registerAction('pause', this.pause_.bind(this));
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    const src = mutations['src'];
+    const controlsList = mutations['controlsList'];
+    const loop = mutations['loop'];
+
+    if (src !== undefined || controlsList !== undefined || loop !== undefined) {
+      this.buildAudioElement();
+      this.loadPromise(this.audio_);
+    }
+
+    if (!this.audio_) {
```

Comment:
Looks like this check should just be at the top of the method.

Can we verify? And add a test for this condition?

Reply:
Actually, I just remembered why I did that. It's `buildAudioElement` that populates `this. audio_`, that's why it's ok to use it before the check for whether it's null or not